db/seeds.d/102-organizations.rb

```diff
@@ -0,0 +1,27 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+#
+# !!! PLEASE KEEP THIS SCRIPT IDEMPOTENT !!!
+#
+
+Organization.all.each do |org|
+  User.current = User.anonymous_admin
+  ForemanTasks.sync_task(::Actions::Katello::Organization::Create, org) unless org.library
+  User.current = nil
+end
+
+if ENV['SEED_ORGANIZATION']
+  Organization.without_auditing do
+    User.current = User.anonymous_admin
+    org = Organization.find_by_name!(ENV['SEED_ORGANIZATION'])
+    ForemanTasks.sync_task(::Actions::Katello::Organization::Create, org) unless org.library
+    User.current = nil
+  end
```

Comment:
Won't the condition starting on line 7 handle this case given that the organization would have been created in an earlier seed ?